xptifw/CMakeLists.txt

```diff
@@ -14,17 +17,28 @@ if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
   set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type (default Release)" FORCE)
 endif()
 
-project (xptifw)
+foreach(flag_var
```

Comment:
Please, add a comment with the reason to make this replacement.

Reply:
Actually, all the public interfaces are wrapped with `extern C`, so this is not needed. Removed.